api/v1/middleware/ratelimiting.js

```diff
@@ -0,0 +1,15 @@
+const rateLimiter = require('express-rate-limit-middleware').rateLimit
+
+const config = require('../../config');
+const redisStorage = require('express-rate-limit-middleware').redisRateLimit
+const redisClient = require('../../cache').instance();
+
+module.exports = function(limitCount = config.ratelimiting.limit, resetTimer = config.ratelimiting.resetTimer) {
```

Comment:
What is the necessity of the top level function?

Reply:
I'd say just use the config values instead of providing the option to override here. No need for a top level function.